onboarding/source/controller.c

```diff
@@ -56,7 +57,18 @@ uint8_t initController(void) {
 
     /* USER CODE BEGIN */
     // Create light timer and check if task/timers were created successfully
+    if(lightTimerHandle == NULL) {
+        // Create light timer
+        lightTimerHandle = xTimerCreate(LIGHT_TIMER_NAME, 
+                                        LIGHT_TIMER_PERIOD, 
+                                        LIGHT_TIMER_AUTORELOAD, 
+                                        (void *) 0, 
+                                        lightTimerCallback);
+    }
 
```

Comment:
you might want to check if the timer failed to create (check if lightTimerHandle is NULL)